sql/catalyst/src/main/scala/org/apache/spark/sql/errors/QueryCompilationErrors.scala

```diff
@@ -2371,4 +2371,8 @@ object QueryCompilationErrors {
     new AnalysisException(
       s"Invalid view text: $viewText. The view $tableName may have been tampered with")
   }
+
+  def invalidTimeTravelSpecError(errorMessage: String): Throwable = {
```

Comment:
Seems like this method does not need a parameter. We can write the error message here.